### PR TITLE
Add Connected Mode menu

### PR DESCRIPTION
**Pull request by rita-sonarsource**
Fixes [#3372](https://github.com/SonarSource/sonarlint-visualstudio/issues/3372)

### CODE REVIEW

File: src/Integration.Vsix/SonarLint.vsct [hunk 1]
```diff
@@ -46,17 +47,13 @@
           <ButtonText>Help</ButtonText>
         </Strings>
       </Menu>
-    </Menus>
-
-    <!-- Example of how to add a command to the sonarlint sub menu
```
Comment (rita-sonarsource) [inline src/Integration.Vsix/SonarLint.vsct:51]: I removed this commented sample because we're already using it, so we have the real buttons as examples.

**Comment by georgii-borovinskikh-sonarsource** [comment]
You only added the menu in this PR. Will other lower level buttons be in another PR?

**Comment by rita-sonarsource** [comment]
> You only added the menu in this PR. Will other lower level buttons be in another PR?

Yep, https://github.com/SonarSource/sonarlint-visualstudio/pull/3362 @georgii-borovinskikh-sonarsource

**Review by bigfluffycookie — APPROVED** [review]
Looks good!

**Pull request merged**